x-pack/plugins/ml/server/shared_services/shared_services.ts

```diff
@@ -173,6 +191,10 @@ function getRequestItemsProvider(
       isMlReady
     );
 
+    const uiSettingsClient = getUiSettings()!.asScopedToClient(savedObjectsClient);
```

Comment:
it might be best to get UI settings and Field Formats earlier, like `clusterClient` above (line 185), and then throw an error if either are null, like line 198 below

Reply:
Updated in https://github.com/elastic/kibana/pull/109073/commits/23b5355ebd3679e42878098ebabbcd9d19fbae73 and https://github.com/elastic/kibana/pull/109073/commits/c263baf35044c7c3f9d69c517ebdbc67403ba0c8